media-libs/x265/x265-3.5.recipe

```diff
@@ -38,7 +38,7 @@ REQUIRES_devel="
 
 PROVIDES_bin="
 	x265${secondaryArchSuffix}_bin = $portVersion
-	cmd:x265 = $portVersion
+	cmd:x265$secondaryArchSuffix = $portVersion
```

Comment:
should be installed $prefix/bin

Reply:
Done.

Reply:
```suggestion
	cmd:x265 = $portVersion
```

Reply:
Wouldn't this brign to [this](https://discuss.haiku-os.org/t/policy-error-from-haikuporter/13230) ?

Reply:
no, the bin is installed in $prefix/bin